src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNetCoreMvc2Integration.cs

```diff
@@ -58,10 +59,10 @@ public AspNetCoreMvc2Integration(object actionDescriptor, object httpContext)
                 }
 
                 string url = GetDisplayUrl(request).ToLowerInvariant();
-
                 SpanContext propagatedContext = null;
+                var tracer = Tracer.Instance;
```

Comment:
Why the change to a local variable? verbosity?

Reply:
Partly as (probably premature) optimization, since it removes several calls to the property getter (and this code is called by the profiler, which disables method inlining).

Partly because `Tracer.Instance` can change, and we should use the same `Tracer` instance throughout this code, even if `Tracer.Instance` changes halfway through.